agent/custom/action/combat.py

```
@@ -883,17 +936,19 @@ def run(
         context.run_task("SSToReplayIfCan")
 
         # 看看要不要吃不吃糖
-        available_count = _tc_get_available_count(context)
-        if available_count == -1:
+        availability = _tc_get_availability(context)
+        available_count = availability.available_count
+        if available_count is None:
             logger.debug("识别战斗次数失败")
             available_count = 1
         elif available_count <= 0:
             logger.debug("没体力咯，吃个糖")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)